modules/packages/TOML.chpl

```diff
@@ -1363,7 +1363,12 @@ module TomlReader {
       }
     }
 
-    proc readWriteThis(f) throws {
+    pragma "no doc"
+    proc readThis(f) throws {
+      compilerError("Reading a Tokens type is not supported");
+    }
+
+    proc writeThis(f) throws {
       // TODO: The `list` type currently doesn't support readWriteThis!
```

Comment:
Should this comment also be updated?

Reply:
I think it's mainly referring to the <~> operator there. I can update it in the other PR.